components/markets/MarketScroll.tsx

```diff
@@ -32,39 +33,27 @@ const MarketScroll = ({
   //calculate cards shown and width based on container width
   const cardsShown =
     windowWidth < BREAKPOINTS.md ? 1 : windowWidth < BREAKPOINTS.lg ? 2 : 3;
+
   const cardWidth =
     windowWidth < BREAKPOINTS.md
       ? containerWidth
       : windowWidth < BREAKPOINTS.lg
       ? (containerWidth - gap) / cardsShown
       : (containerWidth - gap * 2) / cardsShown;
-  const scrollMin = 0;
-  const scrollMax = cardWidth * markets.length + gap * (markets.length - 1);
-
-  const moveSize = cardsShown * (cardWidth + gap);
-
-  useEffect(() => {
-    scrollRef.current?.scroll({ left: scrollLeft, behavior: "smooth" });
-  }, [scrollRef, scrollLeft]);
 
   const handleRightClick = () => {
-    setScrollLeft((prev) => {
-      const newScroll = prev + moveSize;
-      const max = scrollMax - containerWidth;
-      return newScroll > max ? scrollMax - containerWidth : newScroll;
-    });
+    setPageIndex(pageIndex + 1);
   };
 
   const handleLeftClick = () => {
-    setScrollLeft((prev) => {
-      const newScroll = prev - moveSize;
-
-      return newScroll < scrollMin ? scrollMin : newScroll;
-    });
+    setPageIndex(pageIndex - 1);
   };
 
-  const hasReachedEnd = scrollMax - containerWidth - scrollLeft === 0;
-  const leftDisabled = scrollLeft === 0;
+  const pageDelta = markets.length / cardsShown;
```

Comment:
What's the reason for rewriting this? I'm noticing some strange behaviour
<img width="466" alt="image" src="https://github.com/zeitgeistpm/ui/assets/4950844/850a9f54-9220-4288-88c1-639d837beae0">

Reply:
Yeah I had to rewrite a bit because when we use overflow scroll the cards get clipped when hovering and its scale is increased outside the boundaries of the container. So I am using translation to offset and opacity to show them. Its a bit more tricky, but a nice thing about it is that the cards fade in from the edges on bigger screens.

Reply:
Should be fixed